helpers/UserHelper.js

```diff
@@ -10,12 +11,12 @@ const { User } = db;
  * */
 class UserHelper {
   /**
-   * Find a particular user  by emial
-   * @async
-   * @param {string} email - the user email
-   * @return {object} Returns json object
-   * @static
-   */
+     * Find a particular user  by emial
```

Comment:
I noticed you refactored some code and you've changed some of the formattings in the code, I nice job thumbs up. However, can you revert the formatting you did on the JSDoc and keep the previous formatting. Thanks